lib/OpenQA/WebAPI/Controller/Test.pm

```diff
@@ -421,6 +421,8 @@ sub _show {
             show_autoinst_log => $job->should_show_autoinst_log,
             show_investigation => $job->should_show_investigation,
             show_live_tab => $job->state ne DONE,
+            testgiturl => gitrepodir(distri => $job->DISTRI),
+            needlegiturl => gitrepodir(distri => $job->DISTRI, needles => 1),
```

Comment:
Likely my last comment got lost. I still think these URLs should be rendered on the tab's page (and not the overall page).